be/src/exec/parquet_reader.cpp

```diff
@@ -152,6 +166,43 @@ Status ParquetReaderWrap::init_parquet_reader(const std::vector<SlotDescriptor*>
     }
 }
 
+Status ParquetReaderWrap::get_schema(std::vector<std::string>* col_names, std::vector<TypeDescriptor>* col_types) {
+    RETURN_IF_ERROR(_init_parquet_reader());
+
+    auto schema = _file_metadata->schema();
+    for (int i = 0; i < schema->num_columns(); ++i) {
+        auto column = schema->Column(i);
+        auto physical_type = column->physical_type();
+
+        auto name = column->name();
+        TypeDescriptor tp;
+
+        switch (physical_type) {
+        case parquet::Type::BOOLEAN:
+            tp = TypeDescriptor(TYPE_BOOLEAN);
+            break;
+        case parquet::Type::INT32:
+        case parquet::Type::INT64:
```

Comment:
why int32/int64 is LargeInt?

Reply:
Updated. The int32 is mapped as INT, and the int64 is mapped as BIGINT.